api/src/schema/campaigns-equipment.graphql

```diff
@@ -9,8 +9,8 @@ interface EquipmentRecord {
 }
 
 type ConstituencyEquipment implements EquipmentRecord {
-  id: ID!
-  pulpits: Int!
+  id: ID
+  pulpits: Int
```

Comment:
Why would rather take the exclamation marks off?

Reply:
When the fellowships have filled their forms, but the constituency has not, there will be some data. Making the ID and pulpits non nullable will throw an error even though there is data for the constituency